### DISCUSSION TITLE
启动报错

### DISCUSSION THREAD

**Discussion by sandysos** [myshell-ai/AIlice#19] [Q&A]
执行启动命令：ailice_web --modelID=oai:gpt-4-1106-preview --prompt="main" 报错：
We now start the vector database. Note that this may include downloading the model weights, so it may take some time.
Backend MacOSX is interactive backend. Turning interactive mode on.
Encountered an exception, AIlice is exiting: Model name nomic-ai/nomic-embed-text-v1 was not found.
  File "/Users/leo/workspace/AI/AIlice/ailice/AIliceWeb.py", line 219, in main
    mainLoop(**kwargs)
  File "/Users/leo/workspace/AI/AIlice/ailice/AIliceWeb.py", line 59, in mainLoop
    msg = storage.Open("")

**Comment by sandysos** [comment]
@stevenlu137  感谢

**Reply by stevenlu137** [reply]
It seems that HuggingFace failed to load the model. Sometimes hf maintenance can cause such problems. Have you tried it again?
And did this happen after the first fresh installation of AIlice? 
If this problem always occurs, you may also need to check your "梯子" .
